modules/gitops-accessing-the-gitops-operator-metrics.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-the-gitops-operator-performance.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-the-gitops-operator-metrics_{context}"]
+= Accessing the {gitops-shortname} Operator metrics
+
+You can access the Operator metrics from the *Administrator* perspective of the {OCP} web console to track the performance of the Operator.
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default `openshift-gitops-operator` namespace.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Observe* -> *Metrics*.
+
+. Enter the metric in the *Expression* field. You can choose from the following metrics:
+
+* `active_argocd_instances_total`
+* `active_argocd_instances_by_phase`
+* `active_argocd_instance_reconciliation_count`
+* `controller_runtime_reconcile_time_seconds_per_instance_bucket`
+* `controller_runtime_reconcile_time_seconds_per_instance_count`
+* `controller_runtime_reconcile_time_seconds_per_instance_sum`
```

Comment:
In the tool-tip of the web console, we can see the description and type of these metrics as we enter them in the query.

But we are missing that information for `active_argocd_instance_reconciliation_count`. By the definition, it should be a counter but the description is missing in the UI.

Reply:
cc: @jaideepr97
Here is a screenshot to see what we mean. As you can see, the `active_argocd_instance_reconciliation_count` description and type is missing.
![Screenshot from 2023-11-22 11-30-03](https://github.com/openshift/openshift-docs/assets/98941675/7e8fbbea-716e-43af-9b22-a5c7d4078763)

![Screenshot from 2023-11-22 11-34-47](https://github.com/openshift/openshift-docs/assets/98941675/7d4c56a1-31e8-4e7e-aa47-b7e827c9a19f)

Reply:
Please create a bug and assign it to @jaideepr97

Reply:
we don't do anything different while registering these metrics though
it could be a openshift console UI thing because the length of the metric is too long to display the type as well?
I can investigate

Reply:
Sure, @jaideepr97 . Thanks a lot.

Here is the bug I created: https://issues.redhat.com/browse/GITOPS-3665
PTAL. (FYI: @iam-veeramalla)